lib/commands/execute.js

```diff
@@ -43,6 +43,8 @@ extensions.executeMobile = async function executeMobile (mobileCommand, opts = {
 
     uiautomator: 'mobileUiautomator',
 
+    uiautomatorPageSource: 'mobileUiAutomatorPageSource',
```

Comment:
`mobileUiautomatorPageSource` (to align with the style of the above method)

Reply:
fixed